pkg/authorization/controller/authorizationsync/normalize.go

```diff
@@ -1,172 +0,0 @@
-package authorizationsync
-
-import (
-	"strings"
-
-	apiequality "k8s.io/apimachinery/pkg/api/equality"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/kubernetes/pkg/apis/rbac"
-
-	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
-)
-
-// ConvertToRBACClusterRole performs the conversion and guarantees the returned object is safe to mutate.
-func ConvertToRBACClusterRole(originClusterRole *authorizationapi.ClusterRole) (*rbac.ClusterRole, error) {
-	// convert the origin role to an rbac role
-	convertedClusterRole := &rbac.ClusterRole{}
-	if err := authorizationapi.Convert_authorization_ClusterRole_To_rbac_ClusterRole(originClusterRole, convertedClusterRole, nil); err != nil {
-		return nil, err
-	}
-	// do a deep copy here since conversion does not guarantee a new object.
-	equivalentClusterRole := &rbac.ClusterRole{}
-	if err := rbac.DeepCopy_rbac_ClusterRole(convertedClusterRole, equivalentClusterRole, cloner); err != nil {
-		return nil, err
-	}
-
-	// normalize rules before persisting so RBAC's case sensitive authorizer will work
-	normalizePolicyRules(equivalentClusterRole.Rules)
-
-	// there's one wrinkle.  If `openshift.io/reconcile-protect` is to true, then we must set rbac.authorization.kubernetes.io/autoupdate to false to
-	if equivalentClusterRole.Annotations["openshift.io/reconcile-protect"] == "true" {
-		equivalentClusterRole.Annotations["rbac.authorization.kubernetes.io/autoupdate"] = "false"
```

Comment:
if it's already synced, the annotation on the underlying object is already set. did you add this to the conversion logic so the kube annotation surfaces as the origin annotation when the origin resources are requested, and vice versa?